src/backend/launcher.ts

```diff
@@ -789,8 +789,12 @@ async function callRunner(
       signal: abortController.signal
     })
 
-    if (gameInfo && gameInfo.runner === 'hyperplay')
-      openOverlay(gameInfo?.app_name, gameInfo.runner)
+    const shouldOpenOverlay =
+      gameInfo &&
+      (gameInfo.runner === 'hyperplay' ||
+        (gameInfo.runner === 'sideload' && gameInfo.web3?.supported))
+
+    if (shouldOpenOverlay) openOverlay(gameInfo?.app_name, gameInfo.runner)
```

Comment:
this solves https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/598